database/seeders/DepartmentSeeder.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class DepartmentSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $departments = [
+            'Digital Marketing',
+            'Project Management',
```

Comment:
better to maintain these department lists in a config file and retrieve it using `config()` helper function.

This way you can reuse these list anywhere in the application later on while developement